<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page_init</class>
 <widget class="QWidget" name="page_init">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>1920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_init_message">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>920</y>
     <width>891</width>
     <height>471</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>33</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Initializing</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_fail_message">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>1430</y>
     <width>531</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>33</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_continue">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>1410</y>
     <width>251</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>Continue anyways</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_reboot">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>1410</y>
     <width>251</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>Reboot</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
